### DISCUSSION TITLE
Follow window

### DISCUSSION THREAD

**Discussion by mshzsh** [hyprwm/Hyprland#1137] [Q&A]
Is it possible that  for example, when I click on a web link in workstation 3, I will automatically navigate to the workstation that has Firefox in it?
(Like `follow` option in bspwm)

**Comment by JustSimplyKyle** [comment]
`misc:focus_on_activate=true`

**Reply by mshzsh** [reply]
the `focus_on_activate`  is `true` by default. but not working

**Reply by vaxerski** [reply]
it's not, forgot to edit that like a month ago.

**Reply by mshzsh** [reply]
This problem is solved in almost all my programs, but it still exists in `firefox`, of course I use `firefox developer edition`  and it is unlikely that the problem is `hyprland`

**Comment by Rydgel** [comment]
I have the same issue.
Using latest GIT version `4fd90144`
